spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteManifestsSparkAction.java

```diff
@@ -160,6 +164,34 @@ public RewriteManifestsSparkAction stagingLocation(String newStagingLocation) {
     return this;
   }
 
+  /**
```

Comment:
I think we do not need to javadoc the implementation methods. If there is anything we did not mention in the javadoc of the interface, we should add it there.